fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/catalog/FlinkCatalog.java

```diff
@@ -91,7 +91,7 @@
 import static org.apache.flink.util.Preconditions.checkArgument;
 
 /** A Flink Catalog for fluss. */
-public class FlinkCatalog implements Catalog {
+public class FlinkCatalog extends AbstractCatalog {
```

Comment:
Why extending `AbstractCatalog`? I think it's on purpose to support `null` defaultDatabase.